apps/app/src/views/routes/PrescriptionForm.tsx

```diff
@@ -95,6 +95,7 @@ export const PrescriptionForm = () => {
   }, [ref.current, patientId]);
 
   const orgSettings = getSettings(user?.org_id);
+  //  TODO: remove enable-new-medication-search after discovery
```

Comment:
nit: i hate TODOs in code; too easy to overlook! i prefer tickets for tracking what's necessary

Reply:
HA I agree, I guess I could have done a whitespace commit, but TODO: ignore my comment